### PR TITLE
qgis: 3.10.13 -> 3.16.5

### PR DESCRIPTION
**Pull request by lsix**
###### Motivation for this change

The file has been reformated using nixpkgs-fmt.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/gis/qgis/unwrapped.nix [hunk 1]
```diff
@@ -1,37 +1,71 @@
...
+    hdf5
+    netcdf
+    qtbase
+    qtsensors
+    qca-qt5
+    qtkeychain
+    qscintilla
+    qtserialport
+    qtxmlpatterns
+  ] ++ lib.optional withGrass grass ++ lib.optional withWebKit qtwebkit
```
Comment (SuperSandro2000) [inline pkgs/applications/gis/qgis/unwrapped.nix:68]:
```suggestion
  ] ++ lib.optional withGrass grass
  ++ lib.optional withWebKit qtwebkit
```

Comment (lsix) [reply pkgs/applications/gis/qgis/unwrapped.nix]: Done.

**Comment by tomberek** [comment]
Result of `nixpkgs-review pr 119364` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>qgis</li>
    <li>qgis-unwrapped</li>
  </ul>
</details>

**Comment by tomberek** [comment]
My existing projects load without issue (not a comprehensive check, but better than nothing).

**Pull request merged**